packages/postman-to-openapi/src/convert.ts

```diff
@@ -0,0 +1,771 @@
+}
+/* Helper Functions */
+function calculateFolderTag(
+  { tag, name }: { tag: string; name: string },
+  { separator = ' > ', concat = true },
+): string {
+  return tag && concat ? `${tag}${separator}${name}` : name
+}
+
+function compileInfo(postmanJson: any, optsInfo: any): any {
+  const { info = {}, variable = [] } = postmanJson
+  const { name, description: desc } = info
+  const ver = getVarValue(variable, 'version', '1.0.0')
+  const {
+    title = name,
+    description = desc,
+    version = ver,
+    termsOfService,
+    license,
+    contact,
+    xLogo,
+  } = optsInfo
+  return {
+    title,
+    description,
+    version,
+    ...parseXLogo(variable, xLogo),
+    ...(termsOfService ? { termsOfService } : {}),
+    ...parseContact(variable, contact),
+    ...parseLicense(variable, license),
+  }
+}
+
+function parseXLogo(variables: PostmanVariable[], xLogo: any = {}): any {
+  const urlVar = getVarValue(variables, 'x-logo.urlVar')
+  const backgroundColorVar = getVarValue(variables, 'x-logo.backgroundColorVar')
+  const altTextVar = getVarValue(variables, 'x-logo.altTextVar')
+  const hrefVar = getVarValue(variables, 'x-logo.hrefVar')
+  const {
+    url = urlVar,
+    backgroundColor = backgroundColorVar,
+    altText = altTextVar,
+    href = hrefVar,
+  } = xLogo
+  return url != null
+    ? { 'x-logo': { url, backgroundColor, altText, href } }
+    : {}
+}
+
+function parseLicense(
+  variables: PostmanVariable[],
+  optsLicense: any = {},
+): any {
+  const nameVar = getVarValue(variables, 'license.name')
+  const urlVar = getVarValue(variables, 'license.url')
+  const { name = nameVar, url = urlVar } = optsLicense
+  return name != null ? { license: { name, ...(url ? { url } : {}) } } : {}
+}
+
+function parseContact(
+  variables: PostmanVariable[],
+  optsContact: any = {},
+): any {
+  const nameVar = getVarValue(variables, 'contact.name')
+  const urlVar = getVarValue(variables, 'contact.url')
+  const emailVar = getVarValue(variables, 'contact.email')
+  const { name = nameVar, url = urlVar, email = emailVar } = optsContact
+  return [name, url, email].some((e) => e != null)
+    ? {
+        contact: {
+          ...(name ? { name } : {}),
+          ...(url ? { url } : {}),
+          ...(email ? { email } : {}),
+        },
+      }
+    : {}
+}
+
+function parseExternalDocs(
+  variables: PostmanVariable[],
+  optsExternalDocs: any,
+): any {
+  const descriptionVar = getVarValue(variables, 'externalDocs.description')
+  const urlVar = getVarValue(variables, 'externalDocs.url')
+  const { description = descriptionVar, url = urlVar } = optsExternalDocs
+  return url != null
+    ? { externalDocs: { url, ...(description ? { description } : {}) } }
+    : {}
+}
+
+function parseBody(body: any = {}, method: string): any {
+  if (['GET', 'DELETE'].includes(method)) return {}
+  const { mode, raw, options = { raw } } = body
+  let content: any = {}
+  switch (mode) {
+    case 'raw': {
```

Comment:
No `switch` and no `let`. In the scalar codestyles you can see examples that use the early return pattern which is much easier to follow. 

```ts
if(conditionOne) return dataFormOne
if(conditionTwo) return dataFormTwo
```